streams/src/main/java/org/apache/kafka/streams/kstream/ForeachProcessor.java

```diff
@@ -19,7 +19,9 @@
 import org.apache.kafka.streams.processor.api.Processor;
 import org.apache.kafka.streams.processor.api.Record;
 
-// deprecated since 4.0.
+/**
+ * This class is deprecated and should not be used any longer.
```

Comment:
Please keep "since 4.0" as it is important information to remind us the deprecation cycle ...